src/schema/v2/fields/money.ts

```diff
@@ -225,7 +226,7 @@ export const resolveMinorAndCurrencyFieldsToMoney = async (
       major,
       cents: minor,
       currency: currencyCode,
-      display: formatMoney(major, symbolFromCurrencyCode(currencyCode), 0),
+      display: priceDisplayText(minor, currencyCode, ""),
```

Comment:
note this formatting is more in line with our design guidelines but is specific to 'price display'... which may not be what all prices mean. I may follow up with a more specific attempt to address this.